config/default/kustomization.yaml

```diff
@@ -9,8 +9,15 @@ namespace: kyma-system
 namePrefix: istio-
 
 # Labels to add to all resources and selectors.
-commonLabels:
-  app.kubernetes.io/component: istio-operator.kyma-project.io
+
+labels:
+  - includeSelectors: false
+    includeTemplates: true
+    pairs:
+      kyma-project.io/module: istio
+      app.kubernetes.io/version: dev
+      app.kubernetes.io/part-of: istio
+      app.kubernetes.io/name: istio-operator
```

Comment:
Why did we not include `app.kubernetes.io/component: operator`, `app.kubernetes.io/instance: istio-operator-default` here?

Reply:
`app.kubernetes.io/component` defines which part of an architecture the app belongs. It's different based on specific component. I think we should label each app component properly instead of shoving same label to every component.

`app.kubernetes.io/instance` should be set to name of unique operator instance, but oh well...